ochre/Equipment/HVAC.py

```diff
@@ -187,6 +187,8 @@ def __init__(self, envelope_model=None, use_ideal_capacity=None, **kwargs):
         if use_ideal_capacity is None:
             use_ideal_capacity = self.time_res >= dt.timedelta(minutes=5) or self.n_speeds >= 4
         self.use_ideal_capacity = use_ideal_capacity
+        if self.n_speeds >= 4:
+            self.use_ideal_capacity = True
```

Comment:
@jmaguire1 , should we take out this here as well or is it ok to keep it?

Reply:
No, we should keep this as is. All variable speed equipment actually is capable of modulating like "use_ideal_capacity" works.